tests/main.test.ts

```diff
@@ -169,19 +74,23 @@ describe("Action tests", () => {
         },
       },
     }));
+    const mergePullRequest = jest.fn();
+    jest.mock(githubHelpersPath, () => {
+      const actualModule = jest.requireActual(githubHelpersPath) as object;
+      return {
+        __esModule: true,
+        ...actualModule,
+        mergePullRequest,
+      };
+    });
     const run = (await import("../src/action")).run;
     await expect(run()).resolves.toMatchObject({ status: 200 });
-    const pullRequests = await dataSource.getRepository(PullRequest).find();
-    expect(pullRequests).toHaveLength(1);
+    expect(mergePullRequest).not.toHaveBeenCalled();
   });
 
   it("Should close a PR that is past the threshold", async () => {
-    const dataSource = await initializeDataSource(dbName);
-    const pr = new PullRequest();
-    pr.url = htmlUrl;
     const lastActivityDate = new Date();
     lastActivityDate.setDate(new Date().getDate() - 8);
```

Comment:
this is not used anymore, or is it?

Reply:
It is used in the timeline for the event date.